src/CMakeLists.txt

```diff
@@ -532,6 +532,8 @@ if (CGNS_ENABLE_FORTRAN)
 else (CGNS_ENABLE_FORTRAN)
   add_library(cgns_static STATIC ${cgns_FILES})
 endif (CGNS_ENABLE_FORTRAN)
+# Needed to work around a CMake > 3.8 bug on Windows with MSVS and Intel Fortran
```

Comment:
This comment may be misleading: I'm not 100% sure if the bug is unique to Windows/Intel Fortran. A regression was introduced in CMake that triggers this. Effectively, enabling Fortran caused the linker language for `cgns_static` to change (to Fortran I think) due to the `OBJECT` library `cgns_f2c`. This happens in newer CMakes.

@brtnfld if you want me to adjust the comment please let me know.